docs/setup-hybrid-OVNKubernetes-cluster.md

```diff
@@ -57,8 +57,10 @@ spec:
             hostPrefix: 23
         hybridOverlayVXLANPort: 9898
 ```
-
-**Note:** The `hybridClusterNetwork` CIDR cannot overlap with the `clusterNetwork` CIDR.
+#### Attention
+- The `hybridClusterNetwork` CIDR cannot overlap with the `clusterNetwork` CIDR
+- You cannot use Windows Server 2019 LTSC (1809) as it does not have the kernel feature required for using custom VXLAN 
```

Comment:
nit: Maybe?
`You cannot use worker nodes running Windows Server 2019 LTSC...`

Reply:
`worker node` does not apply as the instance will never get fully configured and I did not want to use `Machine` given BYOH is in the pipeline. Moreover I feel it is obvious when calling out an OS version.